packages/pieces/box/src/lib/triggers/index.ts

```diff
@@ -0,0 +1,48 @@
+import { boxRegisterTrigger } from "./register"
+
+const triggerData = [
+  {
```

Comment:
Let's focus on top usecases, I suggest:

- New File
- New Folder
- New Comment

For thew new file can we make the default root directory.

Rename Item Id -> Folder Id and make it optional.
and Item type in that case will always be Folder.

Reply:
defaulting to root folder `Access denied - insufficient permission` 

Any other id works.... This s for new file upload trigger